chemprop/train/run_training.py

```diff
@@ -200,9 +200,15 @@ def run_training(args: TrainArgs,
         else:
             debug(f'Building model {model_idx}')
             model = MoleculeModel(args)
-
+            
+        # Optionally, overwrite weights:
+        if args.checkpoint_frzn is not None:
+            debug(f'Loading and freezing parameters from {args.checkpoint_frzn}.')
+            model = load_frzn_model(model=model,path=args.checkpoint_frzn, current_args=args, logger=logger)     
+        
         debug(model)
-        debug(f'Number of parameters = {param_count(model):,}')
+        debug(f'Number of unfrozen parameters = {param_count(model):,}')
+        debug(f'Total number of parameters = {param_count_all(model):,}')
```

Comment:
This is a nice change to these functions.

Most users will not be using the frozen functionality. Might be nice to make it so that the output reads without reference to frozen if you're not using frozen. Maybe have it inside an if/else statement.